backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/nudger/BpkNudger.kt

```diff
@@ -0,0 +1,91 @@
+/*
+ * Backpack for Android - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.skyscanner.backpack.compose.nudger
+
+import androidx.compose.animation.animateColorAsState
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.widthIn
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.style.TextAlign
+import net.skyscanner.backpack.compose.button.BpkButton
+import net.skyscanner.backpack.compose.button.BpkButtonSize
+import net.skyscanner.backpack.compose.button.BpkButtonType
+import net.skyscanner.backpack.compose.fieldset.BpkFieldStatus
+import net.skyscanner.backpack.compose.fieldset.LocalFieldStatus
+import net.skyscanner.backpack.compose.icon.BpkIcon
+import net.skyscanner.backpack.compose.text.BpkText
+import net.skyscanner.backpack.compose.theme.BpkTheme
+import net.skyscanner.backpack.compose.tokens.BpkSpacing
+import net.skyscanner.backpack.compose.tokens.Minus
+import net.skyscanner.backpack.compose.tokens.Plus
+
+@Composable
+fun BpkNudger(
+  value: Int,
+  onValueChange: (Int) -> Unit,
+  range: IntRange,
```

Comment:
As we've got a separate min & max value property on all platforms let's stay consistent with that

Reply:
We can, although I think we need to align to the developer experience in the first place and have the range in simply more convenient.

Reply:
Alignment across platforms (AND with Figma) also improves developer platform - that's why we're now starting to spec out the API first in Figma before working on the component to ensure we're all aligned.

Reply:
Sliders use Ranges as well. Since most of our widgets will be used by Android (not cross-platform) developers, having API aligned with Compose and BpkSlider seems a better option for me

Reply:
We haven't created slider yet, so what we decide there we can align with here as well. The alignment is something we agreed on as a team, so if there's deviation we should discuss it as a team.

Reply:
It has the same semantic meaning – min and max values, it just handles it in a different way. We still follow the same specs. Deviating for the platform-specific API is fine as long as we keep the same meaning from Figma – and we do that with arguments such as Modifier, InteractionSource, state hoisting etc.